models/src/main/scala/coop/rchain/models/rholangn/parmanager/ProtobufSerialization.scala

```diff
@@ -0,0 +1,66 @@
+package coop.rchain.models.rholangn.parmanager
+
+import cats.Eval
+import com.google.protobuf.{CodedInputStream, CodedOutputStream}
+import coop.rchain.models.rholangn.{ParN, RhoTypeN}
+
+import java.io.{ByteArrayInputStream, ByteArrayOutputStream}
+import scala.util.Using
+
+object ProtobufSerialization {
+
+  @SuppressWarnings(Array("org.wartremover.warts.TryPartial"))
+  def serialize(par: RhoTypeN): Eval[Array[Byte]] = {
+
+    /** Wrapper for protobuf serialization of primitive types. */
+    class ProtobufPrimitiveWriter(output: CodedOutputStream) extends PrimitiveWriter {
```

Comment:
This class connects two _worlds_. Our local code with external library and it's basically the only connection that serialization has with protobuf.

So it should be in a separate folder together with the constructor (apply). 

Another function should provide instantiation of writer and output stream, but not mentioning `Serialization.serialize` because we are now in protobuf folder!